examples/post_training_quantization/coco2017/DETR/evaluation.py

```diff
@@ -0,0 +1,95 @@
+import torch
+import os
+
+import util.misc as utils
+from datasets.coco_eval import CocoEvaluator
+from datasets.panoptic_eval import PanopticEvaluator
```

Comment:
where is datasets?

Reply:
`sys.path.append("./detr")` is added to main.py, so` datasets` used here is `detr.datasets`